### PR TITLE
feat: complete

### PR DESCRIPTION
**Pull request by timoff99**

### CODE REVIEW

File: 6rating.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ uri: "http://api.netflix.com/catalog/titles/movies/70111470",
+ rating: [5.0],
+ bookmark: [{ id: 432534, time: 65876586 }]
+}];
+
+const rating = (arr) => {
+  return arr.filter(({rating})=> {
+    return rating[0] === 5
+  }).map(({id}) => id)
+}
```
Comment (adryianSiarheyeu) [inline 6rating.js:37]: it could be implemented through the single reduce method to avoid looping twice

File: 7square.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  {
+    width: 425,
+    height: 150,
+    url: "http://cdn-0.nflximg.com/images/2891/Fracture425.jpg"
+  }
+];
+
+
+const square = (arr) => {
+  const arrOfSquareArr = arr.reduce((prev, cur, index) => {
```
Comment (adryianSiarheyeu) [inline 7square.js:28]: strange name arr of ... arr

**Review by adryianSiarheyeu — CHANGES_REQUESTED** [review]

File: 1cutString.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const cutString = (str, length) => {
+  const newWorld = str.substring(0, length);
+  return `${newWorld}...`
```
[the remark this answers is not in the document]
Comment (timoff99) [reply 1cutString.js:3]: I didn't understand what the problem is

File: 2maxFrequent.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const maxFrequent = (arr) => {
+  const hash = new Map();
+  arr.reduce((prev, cur)=> {
+    prev[cur] = (prev[cur] || 0) + 1;
+    return hash.set(cur, prev[cur]);
+  }, {})
```
[the remark this answers is not in the document]
Comment (timoff99) [reply 2maxFrequent.js:6]: I don't know how to do without `Map`

**Pull request merged**